apps/studio/components/interfaces/Account/AccessTokens/TokenBasicInfoForm.tsx

```diff
@@ -0,0 +1,133 @@
+import { Control } from 'react-hook-form'
+import { useState, useEffect } from 'react'
+import { format } from 'date-fns'
+import { Calendar as CalendarIcon } from 'lucide-react'
+import {
+  FormField_Shadcn_,
+  FormControl_Shadcn_,
+  Select_Shadcn_,
+  SelectTrigger_Shadcn_,
+  SelectValue_Shadcn_,
+  SelectContent_Shadcn_,
+  SelectItem_Shadcn_,
+  Calendar,
+  Popover_Shadcn_,
+  PopoverContent_Shadcn_,
+  PopoverTrigger_Shadcn_,
+  Button,
+  Input_Shadcn_,
+  Input,
```

Comment:
Ended up using this deprecated component due to our newer Input not gracefully handling the actions within the input, maybe something we should revisit in a separate ticket/thing @SaxonF ?